be/src/olap/rowset/segment_v2/column_reader.h

```diff
@@ -188,5 +191,39 @@ class FileColumnIterator : public ColumnIterator {
     rowid_t _current_rowid = 0;
 };
 
+// This iterator is used to read default value column
+class DefaultValueColumnIterator : public ColumnIterator {
+public:
+    DefaultValueColumnIterator(const TabletColumn& tablet_column)
+        : _tablet_column(tablet_column) { }
+
+    ~DefaultValueColumnIterator() { }
+
+    Status init() override;
+
+    Status seek_to_first() override {
+        _current_rowid = 0;
+        return Status::OK();
+    }
+
+    Status seek_to_ordinal(rowid_t ord_idx) override {
+        _current_rowid = ord_idx;
+        return Status::OK();
+    }
+
+    Status next_batch(size_t* n, ColumnBlock* dst) override;
+
+    rowid_t get_current_ordinal() const override { return _current_rowid; }
+
+private:
+    const TabletColumn& _tablet_column;
```

Comment:
Get what this class needs, It's better not to depends TabletColumn.

Reply:
ok, I will optimize this